models/reviews.models.js

```diff
@@ -35,3 +35,21 @@ exports.updateVotes = (reviewId, incVotes) => {
       }
     });
 };
+
+// ALMOST WORKS BUT JUST MISSING THE comment_count KEY.
```

Comment:
Remove please!

Reply:
Already removed before pushing so not sure why its here!